### PR TITLE
Fix some docblocks.

### PR DESCRIPTION
**Pull request by lucasmichot**
- [ ] Added or updated tests
- [ ] Added Docs for all relevant versions

**Related Issue/Intent**

Fix some docblocks.

### CODE REVIEW

File: tests/Integration/Defer/DeferTest.php [hunk 1]
```diff
@@ -303,7 +303,7 @@ public function itCancelsDefermentAfterMaxExecutionTime(): void
         }
         ";
 
-        /** @var \Nuwave\Lighthouse\Defer\Defer $defer */
+        /* @var \Nuwave\Lighthouse\Defer\Defer $defer */
```
Comment (spawnia) [inline tests/Integration/Defer/DeferTest.php]: Docblocks start with `/**`

Comment (spawnia) [reply tests/Integration/Defer/DeferTest.php]: http://docs.phpdoc.org/guides/docblocks.html#anatomy-of-a-docblock

Comment (lucasmichot) [reply tests/Integration/Defer/DeferTest.php]: When they are multiline, not one a single line

Comment (lucasmichot) [reply tests/Integration/Defer/DeferTest.php]: I can change that though

Comment (spawnia) [reply tests/Integration/Defer/DeferTest.php]: Did you look at the link?

Comment (lucasmichot) [reply tests/Integration/Defer/DeferTest.php]: Yes, but I've been tricked by `createNodeResolver ` 😞

Comment (lucasmichot) [reply tests/Integration/Defer/DeferTest.php]: Done : https://github.com/nuwave/lighthouse/pull/664/commits/8ec24bb632726e667920144067c6896e48604825

Comment (lucasmichot) [reply tests/Integration/Defer/DeferTest.php]: Seems this still breaks StyleCI: https://github.styleci.io/analyses/z9j7R6?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Pull request merged**